cmd/fleet/handleEnroll.go

```diff
@@ -293,12 +293,22 @@ func createFleetAgent(ctx context.Context, bulker bulk.Bulk, id string, agent mo
 }
 
 func generateAccessApiKey(ctx context.Context, client *elasticsearch.Client, agentId string) (*apikey.ApiKey, error) {
-	return apikey.Create(ctx, client, agentId, "", []byte(kFleetAccessRolesJSON))
+	return apikey.Create(ctx, client, agentId, "", []byte(kFleetAccessRolesJSON),
+		apikey.Metadata{
+			Application: apikey.FleetAgentApplication,
```

Comment:
Naming: Should we call this application? Service? ...

Reply:
naming is hard. the original ES ticket used "application" https://github.com/elastic/elasticsearch/issues/48182
easy to change, I have no preference. let me know.

Reply:
@tvernum I think you were the one using application there. Any preference?